### PR TITLE
Limit children

### PR DESCRIPTION
**Pull request by nhunzaker**
Allows children to be limited inside a block type with `maxChildren`.

limiter

### CODE REVIEW

File: src/components/Switch.jsx [hunk 1]
```diff
@@ -46,13 +48,27 @@ module.exports = React.createClass({
                        onExit={ this.close } />)
   },
 
+  hasMaxChildren() {
+    let { app, parent } = this.props
+
+    if (!parent) return false
```
Comment (dce) [inline src/components/Switch.jsx]: Does this mean we can't constrain the number of root-level blocks?

Comment (nhunzaker) [inline src/components/Switch.jsx]: Right now no. This is sort of a design flaw in CK right now... I'll have to add it as a separate feature.

I'm starting to think that the root level editor for Colonel Kurtz should be a block itself. Then you could add the same options to blocks as the editor. Right now, that's not the case :-/

Comment (dce) [inline src/components/Switch.jsx]: OK. I'll need to run a couple things we're doing in NTHP by you to see if the best approach is to further update CK or just change my block definitions.

**Comment by cwmanning** [comment]
:+1:

**Pull request merged**